install-onlyoffice.sh

```diff
@@ -143,7 +174,17 @@ install_version() {
     local LAST_DIR
     LAST_DIR=$(pwd)
 
-    if [ ! -e "$FULL_DIR"/.commit ] || [ "$(cat "$FULL_DIR"/.commit)" != "$COMMIT" ]; then
+    local ACTUAL_COMMIT
+    if [ -e "$FULL_DIR"/.commit ]; then
+        ACTUAL_COMMIT="$(cat "$FULL_DIR"/.commit)"
+    fi
+
+    if [ "$ACTUAL_COMMIT" != "$COMMIT" ]; then
```

Comment:
When no OnlyOffice is installed, the script fails here, because `ACTUAL_COMMIT` is unset.